static/app/views/performance/transactionSummary/transactionTags/tagsDisplay.tsx

```diff
@@ -99,15 +97,9 @@ export const TAGS_TABLE_COLUMN_ORDER: TagsTableColumn[] = [
 ];
 
 const TagsDisplay = (props: Props) => {
-  const {eventView: _eventView, location, organization, projects, tagKey} = props;
+  const {eventView: _eventView, location, organization, aggregateColumn, tagKey} = props;
   const eventView = _eventView.clone();
 
-  const aggregateColumn = getTransactionField(
-    SpanOperationBreakdownFilter.None,
-    projects,
-    eventView
-  );
-
```

Comment:
This looked like a straight copy of the code above: https://github.com/getsentry/sentry/pull/36879/files#diff-99a34ffdc3ef6c0e5234dabb63683deca366c1e18c84d9e1bb93803712966da4L46-L49 so I deemed it safe to remove, and pass the aggregate column down as a prop instead. Correct me if I'm wrong!

Reply:
Yep, no reason to repeat it 👍